site/docs/foundations/index.mdx

```diff
@@ -20,30 +30,40 @@ Density relates to the amount of content that can fit on a screen based on the s
   </OverviewCard>
 
 <OverviewCard
-    icon={<Image src="/img/foundations/duration.svg" alt="duration"  />}
+    icon={<Image src="/img/foundations/duration2.svg" alt="duration"  />}
```

Comment:
do you meant to replace the existing svg? if so, add `*2` will leave unused files behind